src/coprocessor/endpoint.rs

```diff
@@ -24,7 +24,7 @@ use tipb::schema::ColumnInfo;
 use protobuf::{CodedInputStream, Message as PbMsg};
 use kvproto::coprocessor::{KeyRange, Request, Response};
 use kvproto::errorpb::{self, ServerIsBusy};
-use kvproto::kvrpcpb::{CommandPri, IsolationLevel};
+use kvproto::kvrpcpb::{CommandPri, HandleTime, IsolationLevel, RunTimeDetails};
```

Comment:
seem runtime is a word, so you can use Runtime. 

But I think naming runtime here seems a little strange.

Reply:
Any suggestion? I'd like to make a new pull request to project kvproto, the previous PR is https://github.com/pingcap/kvproto/pull/217. @siddontang

Reply:
ExecutionDetails?